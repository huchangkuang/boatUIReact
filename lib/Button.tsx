import React, {FC} from 'react';

type Props = {

};
const Button: FC<Props> = (props: Props) => {
  return (
    <div>
      Button
    </div>
  );
};
export default Button