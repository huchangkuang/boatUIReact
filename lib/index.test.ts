import {describe, expect, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import Button from "./Button";

describe('button', () => {
  it('should be div ', () => {
    const component = renderer.create(<Button />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});