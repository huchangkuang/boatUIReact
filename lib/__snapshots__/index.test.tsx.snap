// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button should be div  1`] = `
<div>
  Button
</div>
`;
